docs/tutorial.md

```diff
@@ -23,13 +23,15 @@ _Note: some of the screen images below show the "npm" command. react_on_rails 6.
 
 ##Setting up the environment
 
-Trying out **React on Rails** is super easy, so long as you have the basic prerequisites. This includes the basics for Rails 4.x and node version 5+. I recommend `rvm` and `nvm` to install Ruby and Node. Rails can be installed as ordinary gem.
+Trying out **React on Rails** is super easy, so long as you have the basic prerequisites. This includes the basics for Rails 4.x and node version 6+. I recommend `rvm` and `nvm` to install Ruby and Node, and [brew](https://brew.sh/) to install [yarn](https://yarnpkg.com/en/docs/install#mac-tab). Rails can be installed as ordinary gem.
```

Comment:
May be it will be better to write
> and **npm** to install yarn

'cause it is crossplatform.
I do not have brew on Linux.